mobile-app/app/screens/AppNavigator/screens/Dex/components/UnitSuffix.tsx

```diff
@@ -0,0 +1,30 @@
+import BigNumber from 'bignumber.js'
+
+interface UnitSuffixProps {
+  value?: number | string
+  units: Record<number, string>
+  noSuffixSpacing?: boolean
+  prefix?: string
+}
+
+export function UnitSuffixPrefix (props: UnitSuffixProps): JSX.Element {
```

Comment:
- Let's leave Prefix out from this function, and keep the function to only have a single responsibility, i.e.: to add unit as suffix
- Change to a hook, i.e.: `useUnitSuffix()`
- Should return a string
- Move component to under `hook` folder
- Remove `noSuffixSpacing` since we don't need it now